test/test_core.py

```diff
@@ -3743,6 +3743,8 @@ def test_dlfcn_asyncify(self):
   @requires_jspi
   @needs_dylink
   def test_dlfcn_jspi(self):
+    if self.get_setting('MEMORY64'):
+      self.skipTest('fails with MEMORY64')
```

Comment:
Could you open a bug for that and use the bug URL as the argument to self.skipTest?

Reply:
Sounds good. #23588 seems related so it would make sense to test again after merging that.

Reply:
The error is `LinkError: WebAssembly.instantiate(): cannot import i64 memory as i32` so I wonder if the problem might just be that I don't pass `-sMEMORY64` when linking the side module.

Reply:
Okay I fixed that and got a more real error which does sound like what @brendandahl is already working on.